h2o-automl/src/main/java/ai/h2o/automl/leaderboard/LeaderboardColumn.java

```diff
@@ -0,0 +1,16 @@
+package ai.h2o.automl.leaderboard;
+
+import hex.Model;
+import water.Freezable;
+import water.Key;
+
+public interface LeaderboardColumn<V, SELF extends LeaderboardColumn> extends Freezable<SELF> {
+    LeaderboardColumnDescriptor getDescriptor();
+
+    Key<Model> getModelId();
+    V getValue();
+    void setValue(V value);
+
+    default boolean isNA() { return getValue() == null; }
+    default V fetch() { return getValue(); }
```

Comment:
@seb-h2o not clear when we should use `fetch` and when `getValue`. Maybe naming could be improved?

Reply:
`getValue` is an accessor, by convention in Java, accessors are not supposed to do long running tasks, user should be safe to call them anytime.
`fetch` explicitly says that something needs to be retrieved.
I will add Javadoc, but I think the naming is quite standard.